### PR TITLE
Handle page element insertion and iframe url update

### PR DESCRIPTION
**Pull request by codesource**
* Initialize content inserted after page initialization (ajax call)
* Allow iframe url update by scripting so when iframe is refreshed we have the right url.

### CODE REVIEW

File: Resources/Public/JavaScript/Editor.js [hunk 1]
```diff
@@ -71,114 +71,125 @@ define(['jquery', 'ckeditor', 'ckeditor-jquery-adapter'], function ($, CKEDITOR)
 
 		// Find all t3-frontend-editing__inline-actions
 		var $inlineActions = $iframeContents.find('span.t3-frontend-editing__inline-actions');
-		$inlineActions.each(function (index) {
-			var that = $(this);
+		var initializeInlineAction = function (index) {
+            var that = $(this);
+            if(!that.data('t3-frontend-editing-initialized')) {
```
Comment (MattiasNilsson) [inline Resources/Public/JavaScript/Editor.js]: Add space between if and start of condition.

File: Resources/Public/JavaScript/Editor.js [hunk 2]
```diff
@@ -71,114 +71,125 @@ define(['jquery', 'ckeditor', 'ckeditor-jquery-adapter'], function ($, CKEDITOR)
...
+                } else {
+                    that.find('.icon-actions-move-down').hide();
+                }
+                that.data('t3-frontend-editing-initialized', true);
+            }
+        };
+		$inlineActions.each(initializeInlineAction);
+		// Make sure that elements inserted after page initialization are initialized.
+        $iframeContents.on('mouseover', 'span.t3-frontend-editing__inline-actions', function(){
+            //TODO: Find a clean solution to update move up/move down action
```
Comment (MattiasNilsson) [inline Resources/Public/JavaScript/Editor.js]: Make space between comment and // TODO

**Comment by codesource** [comment]
The requested changes have been done.

**Comment by MattiasNilsson** [comment]
Awesome and thanks for the PR!

**Pull request merged**